### ISSUE TITLE
Windows Server 2003 compatability

### ISSUE THREAD

**Issue by Zero3K** [zodiacon/PoolMonXv3#3]
Please make it compatible with Windows Server 2003 and up.

**Comment by zodiacon** [comment]
I assume the Server 2003 is 64 bit?
What is the exception you're seeing?

**Comment by Zero3K** [comment]
It is 32 bit.

**Comment by zodiacon** [comment]
Try this: 
[PoolMonX.zip](https://github.com/zodiacon/PoolMonXv3/files/13055439/PoolMonX.zip)

**Comment by Zero3K** [comment]
PoolMonX.exe is not a valid Win32 program.

**Comment by Zero3K** [comment]
I tried compiling it for Visual Studio 2017 but I ran into errors.

**Comment by zodiacon** [comment]
Unfortunately, I may be using some stuff that is supported on Vista+ only. I don't have a Server 2003 VM to even test it properly.

**Comment by zodiacon** [comment]
The compilation fails because I'm using VCPckg for most dependencies.

**Comment by Zero3K** [comment]
I was able to install the x86 versions of the dependencies.

**Comment by Zero3K** [comment]
![image](https://github.com/zodiacon/PoolMonXv3/assets/6194332/e517b1f2-3f5d-45ad-856c-28e30662d4f1)

**Comment by Zero3K** [comment]
I also commented out the inclusion of dontuse.h.

**Comment by zodiacon** [comment]
The project is set to C++ 20 (not supported in VS 2017)
Also the XP compiler toolset must be installed.

**Comment by Zero3K** [comment]
Yes, I set it to use C++ 17 instead and have the VS 2017 toolkit installed.

**Comment by zodiacon** [comment]
I am using some C++20 features...

**Comment by zodiacon** [comment]
You can try with VS 2022 with the XP tools and it should work.

**Comment by Zero3K** [comment]
Like what? Also, I do not see the use of the HTML headers anywhere.

**Comment by Zero3K** [comment]
VS 2022 does not include the XP toolkit anymore.

**Comment by zodiacon** [comment]
You're right, sorry...

**Comment by Zero3K** [comment]
It would be nice if you could fix it so it can compile using the VS 2017 toolkit.

**Comment by zodiacon** [comment]
I'm sorry, this requires too many changes. I am just a guy writing these tools in my spare time. I am unable to deal with edge cases like Server 2003.

**Comment by Zero3K** [comment]
Maybe someone could make a tool similar to it. The only other one is one that isn't open source (it is PoolTag from https://www.osronline.com/article.cfm%5Earticle=98.htm).

**Comment by Zero3K** [comment]
Or someone else could get it working on Windows Server 2003.
